src/components/taker/TakerInitialization.js

```
@@ -99,32 +116,35 @@ export function TakerInitializationComponent(container, config, onInitialized) {
+        textEl.classList.add('is-error');
         break;
       default:
-        icon.className += ' bg-gray-600';
-        const num = stepId.includes('tor') ? '1' : stepId.includes('taker') ? '2' : '3';
-        icon.innerHTML = `<span class="text-xs text-white">${num}</span>`;
-        textEl.className = 'text-gray-400 text-sm';
+        const num = stepId.includes('tor')
+          ? '1'
+          : stepId.includes('taker')
+            ? '2'
+            : '3';
```

Biome (2.4.16)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)